lib/generators/init-generator.js

```diff
@@ -50,7 +50,7 @@ module.exports = class InitGenerator extends Generator {
 		process.stdout.write(
 			"\n" +
 				logSymbols.info +
-				chalk.blue(" INFO ") +
+				chalk.blue(" info ") +
```

Comment:
no need

Reply:
I like how yarn shows the message type in lowercase. Again, we can do either way. We just need to follow same consistency for all types of messages.
 
![image](https://user-images.githubusercontent.com/5961873/38558651-6b346608-3cee-11e8-929e-2f89fd1d544e.png)

Reply:
could you src one with the old info text too?